assets/javascripts/admin_assets.js

```diff
@@ -117,11 +120,15 @@ function deleteAsset(assetId) {
     $.ajax({
         url: '/api/v1/assets/' + assetId,
         method: 'DELETE',
+	dataType: 'json',
         success: function() {
-            $('#asset_' + assetId).remove();
+	    var table = $("#assets").DataTable();
+	    table.row($("#asset_" + assetId)).remove().draw();
+	    addFlash('info', 'The asset was deleted successfully. Because of the asset cache, the data are shown in the table may be not effective. To make sure the data are effective, please click "Trigger asset cleanup" button.');
```

Comment:
I would rephrase it as:

> The asset was deleted successfully. The asset table's contents are cached. Hence the removal is not immediately visible. To update the view use the "Trigger asset cleanup" button. Note that this is an expensive operation which might take a while.

Reply:
I would say we do not even know if the actual deletion was successful. We only know that it was triggered, isn't it?

Reply:
We know that is was successful.